api/src/pcapi/core/users/models.py

```diff
@@ -1021,3 +1021,11 @@ def expirationDate(cls):  # pylint: disable=no-self-argument
     @property
     def is_expired(self) -> bool:
         return datetime.utcnow() > self.expirationDate
+
+
+class GdprUserAnonymization(PcObject, Base, Model):
+    __tablename__ = "gdpr_user_anonymization"
+
+    dateCreated: datetime = sa.Column(sa.DateTime, nullable=False, default=datetime.utcnow)
```

Comment:
Peut-être ajouter un `server_default` ?
```suggestion
    dateCreated: datetime = sa.Column(sa.DateTime, nullable=False, default=datetime.utcnow, server_default=func.now())
```